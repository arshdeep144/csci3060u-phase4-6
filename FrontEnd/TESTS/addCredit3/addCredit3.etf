10 admin02 AD 9000
06 user03 999
00 admin02 AD 9000
