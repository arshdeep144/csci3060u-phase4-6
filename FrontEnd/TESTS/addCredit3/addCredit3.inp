login
admin02
addcredit
user03
999
logout