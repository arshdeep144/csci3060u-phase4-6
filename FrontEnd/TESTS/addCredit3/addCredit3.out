Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Welcome to the main menu. Please enter a command to continue: 
Please enter a username
Please enter how much credit you would like to add
Adding credits...
Subtracting credits...
New credits for Admin are: 9000 and new credits for user are: 5946
Welcome to the main menu. Please enter a command to continue: 
